packages/generators/remove-generator.js

```diff
@@ -1,3 +1,46 @@
 const Generator = require("yeoman-generator");
+const { List } = require("@webpack-cli/webpack-scaffold");
 
-module.exports = class RemoveGenerator extends Generator {};
+const PROP_TYPES = require("@webpack-cli/utils/prop-types");
+
+/**
+ *
+ * Generator for removing properties
+ * @class	RemoveGenerator
+ * @extends	Generator
+ * @returns	{Void} After execution, transforms are triggered
+ *
+ */
+
+module.exports = class RemoveGenerator extends Generator {
+	constructor(args, opts) {
+		super(args, opts);
+		this.configuration = {
+			config: {
+				webpackOptions: {},
+			}
+		};
+	}
+
+	prompting() {
+		const done = this.async();
+
+		return this.prompt([
+			List(
+				"actionType",
+				"What property do you want to remove?",
+				Array.from(PROP_TYPES.keys())
+			)
+		])
+			.then(actionTypeAnswer => {
```

Comment:
You'll need to prompt for sub-props too. Check out how the add generator is built

Reply:
I think in case of removal, sub-props would need webpack config to be read. For instance, in case of `plugins`, before we prompt user which plugin he wants to remove, we need to read all plugins from the webpack config file.

In case of add, it works as we prompt all options (or plugins that can be installed) and then the user selects which plugin to install.

If we want to go for sub-prompts then it would need the reading of webpack config file. That's why I've kept it simple for now and removed the key that is selected directly.

Reply:
could you try to impl the read from config? It's quite nice